README.md

```diff
@@ -10,36 +10,72 @@ We welcome questions/comments/feedback (and pull requests).
-- setting up a 2 node Linux cluster either
-  - using Vagrant (on Virtual Box or VMware) for easy setup on your laptop or other virtual environment
-  - by manually setting up your own cluster
-- setting up Calico on the cluster
-- creating containers using Calico networking.
+To get started using [Calico Docker networking][calico-networking], we recommend running through one or more of the
+available tutorials described below.
 
-In addition to the [Getting Started Guide](docs/GettingStarted.md) we also have tutorials for:
-- [Calico on Amazon](docs/AWS.md)
-- [Calico on Google Compute Engine](docs/GCE.md)
-- [Calico on DigitalOcean](docs/DigitalOcean.md)
-- [Calico and Docker Swarm](docs/CalicoSwarm.md)
-- [Calico and Kubernetes](docs/kubernetes/README.md)
+If you would like to get involved writing code for calico-docker, or if you need to build binaries specific to your
+OS, checkout out the [Building and testing guide](docs/Building.md).
 
-Finally, if you want to try out networking containers using Calico with Docker's new [libnetwork network driver support](https://github.com/docker/libnetwork) then you can try out our [Ubuntu Vagrant libnetwork example](https://github.com/Metaswitch/calico-ubuntu-vagrant).
+### Tutorials
+
+Fully worked demonstrations of Calico Docker networking are available for the following environments:
+
+- [Vagrant install with CoreOS](docs/getting-started/VagrantCoreOS.md)
+- [Vagrant install with Ubuntu](docs/getting-started/VagrantUbuntu.md)
+- [Amazon Web Services (AWS)](docs/getting-started/AWS.md)
+- [Google Compute Engine (GCE)](docs/getting-started/GCE.md)
+- [DigitalOcean](docs/getting-started/DigitalOcean.md)
+
+For each of these environments, it is possible to run with one of three different 
+[networking options](#networking-options), described below.  Each of the demonstrations indicates where the instructions
+differ for each of these networking options.
+
+We also provide a demonstration of a [Docker Swarm cluster](docs/getting-started/powerstrip/CalicoSwarm.md) networked 
+using Calico. This particular demonstration utilizes the [powerstrip](#docker-with-powerstrip) networking option.
+
+For a quick start, we'd recommend following one of the Vagrant installations using the default docker networking.
+
+
+### Networking options
+
+#### Default docker networking
+
+This uses Dockers standard networking infrastructure, requiring you to explicitly add a created contained into a
+Calico network.
+
+#### Docker with libnetwork
+
+Docker's native [libnetwork network driver](https://github.com/docker/libnetwork) is available in the Docker
+[experimental channel](https://github.com/docker/docker/tree/master/experimental) alongside the Docker 1.7 release.
+Docker's experimental channel is still moving fast and some of its features are not yet fully stable.  The current

```

Comment:
We _could_ mention that setup is a little more involved as it requires the experimental docker version and a separate datastore to cluster docker.

I also don't think it's worth saying that master of calico-docker is focussed on libnetwork - we're planning to pull the libnetwork stuff out into a new repo soon...